addons/account/models/account_move.py

```diff
@@ -3991,6 +3991,9 @@ def _cron_account_move_send(self, job_count=10, with_commit=True):
 
         for wizard in to_process[:job_count]:
             move_to_lock = wizard.move_ids
+            if not move_to_lock or move_to_lock.state != 'posted':
```

Comment:
`move_to_lock` may be a multirecord, you should use something like `move_to_lock.filtered(lambda m: m.state != 'posted')`

Reply:
Done. :+1: